main.tf

```diff
@@ -239,3 +269,14 @@ resource "azurerm_monitor_diagnostic_setting" "diagnostic_settings" {
     }
   }
 }
+
+
+resource "null_resource" "root_ca" {
```

Comment:
Probably need to update this - https://github.com/hmcts/terraform-module-apim-application-gateway#resources

Reply:
sure -done, added to the readme file.